src/wasm-ops/utils/broadcast_utils.cpp

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+#include "broadcast_utils.h"
+
+std::vector<int32_t> BroadcastUtils::broadcasted_to_original_indices(
+    const std::vector<int32_t> &broadcasted_indices,
+    const std::vector<int32_t> &dims) {
+  const auto rank = static_cast<size_t>(dims.size());
```

Comment:
I don't see the usage of those `static_cast` necessary... is it the compiler complaining about that?